src/components/ProfilePage/QuestCardCompleted/QuestCardCompleted.style.ts

```diff
@@ -68,31 +64,34 @@ export const QuestPlatformMainBox = styled(Box, {
   alignItems: 'center',
 }));
 
+export const QuestDatesBox = styled(Box)(({ theme }) => ({
+  display: 'flex',
+  alignItems: 'center',
+  backgroundColor:
+    theme.palette.mode === 'light'
+      ? alpha(theme.palette.black.main, 0.04)
+      : theme.palette.alphaLight300.main,
+  paddingTop: '4px',
+  paddingBottom: '4px',
+  paddingLeft: '8px',
+  paddingRight: '8px',
+  borderRadius: '128px',
+  justifyContent: 'center',
+}));
+
 export interface XPDisplayBoxProps extends Omit<BoxProps, 'component'> {
   active?: boolean;
 }
 
 export const XPDisplayBox = styled(Box, {
   shouldForwardProp: (prop) => prop !== 'active',
 })<XPDisplayBoxProps>(({ active }) => ({
+  width: active ? '50%' : '100%',
   marginRight: active ? '8px' : undefined,
   display: 'flex',
-  height: '28px',
-  alignContent: 'center',
-  justifyContent: 'space-between',
+  height: '40px',
   alignItems: 'center',
   borderRadius: '128px',
-  padding: '8px',
-}));
-
-export const XPIconBox = styled(Box)(({ theme }) => ({
-  display: 'flex',
-  alignContent: 'flex-end',
-  justifyContent: 'flex-end',
-}));
-
-export const OPBadgeRelativeBox = styled(Box)(({ theme }) => ({
-  position: 'relative',
-  bottom: '16px',
-  right: '32px',
+  padding: '6px',
```

Comment:
```suggestion
  padding: theme.spacing(0.75),
```

Reply:
why sometimes using theme.spacing and sometimes just removing "px"?

Reply:
Good point, I messed up here, but I´d go with `theme.spacing`.

We can discuss in the team, if we can agree on a certain approach: For me properties like margin, padding, gap are related to spacings. Try to avoid `theme.spacing` on any styles like borderRadius, height, width, whatsoever...